README.md

```diff
@@ -5,20 +5,14 @@ imaged exoplanets: a simple archive of complex objects. The tables and fields cu
 database are described in the [Documentation](documentation/README.md) 
 and currently include names, coordinates, photometry and reference and data provenance information and is visualized 
 in the [schema](#simple-database-schema) below. 
-We are currently working on including kinematics, spectra, images, and modeled and retrieved parameters. 
+The archive includes kinematics, spectra, modeled and retrieved parameters. 
 
-We are developing several different methods to interact with the database, including python, a website and API, and 
-database browsers.
+There are several different methods to interact with the database, including: 
+- Python
+- a website and API at https://simple-bd-archive.org/
+- database browsers such as [DB Browser for SQLite](https://sqlitebrowser.org/)
 
-While we are using brown dwarfs to build out the SIMPLE database, our intention is to build a database schema and 
-software which could be used by other subfields to roll their own collaborative databases and web interfaces.
-
-This database uses the [SQLAlchemy ORM](https://docs.sqlalchemy.org/en/14/orm/index.html) and is designed to be
-interacted with via the `astrodbkit2` package.
-
-If you'd like to participate or just stay in the loop as this project progresses, please request to join this discussion
- list:
-https://groups.google.com/forum/#!forum/simple-archive
+To report missing data, data bugs or make feature requests, please use the [SIMPLE-db issue tracker](https://github.com/SIMPLE-AstroDB/SIMPLE-db/issues).
```

Comment:
do we want to point directly to some issue templates as well?

Reply:
Not yet, most of the templates use old methods.

Reply:
When they click New issue, they'll get an option for the templates. I don't think it's necessary here.